x-pack/plugins/code/server/indexer/index_migrator.ts

```diff
@@ -57,9 +58,18 @@ export class IndexMigrator {
           body,
         });
       } catch (error) {
-        this.log.error(`Create new index ${newIndexName} for index migration error.`);
-        this.log.error(error);
-        throw error;
+        if (
+          error.body &&
+          error.body.error &&
+          error.body.error.type === EsException.RESOURCE_ALREADY_EXISTS_EXCEPTION
+        ) {
+          this.log.info(`The target verion index already exists. Skip index migration`);
```

Comment:
maybe use debug, since it doesn't provide useful information to user

Reply:
done